static/app/views/discover/eventDetails/linkedIssue.tsx

```diff
@@ -12,6 +12,7 @@ import ShortId from 'sentry/components/shortId';
 import GroupChart from 'sentry/components/stream/groupChart';
```

Comment:
I don't believe this component is actually being used anywhere, but out of an abundance of caution, I have still refactored this instance of groupChart

Reply:
Hmm it looks like there must have been a recent change that redirects from the discover event details to the issue details page, but I don't know where that redirect is happening or why it was added. Probably why you can't find this anywhere.